app/src/main/java/com/kms/cura/view/fragment/Patient_Home_Fragment.java

```diff
@@ -43,7 +39,7 @@
 import java.util.List;
 import java.util.Locale;
 
-public class PatientHomeFragment extends Fragment implements RadioGroup.OnCheckedChangeListener, UpdateSpinner, ReloadData {
+public class Patient_Home_Fragment extends Fragment implements RadioGroup.OnCheckedChangeListener, UpdateSpinner, ReloadData {
```

Comment:
Don't rename class like this. Change it back to PatientHomeFragment. Refer to Oracle naming convention for Java